tests/unit/json/pull_review_example

```diff
@@ -0,0 +1,36 @@
+{
+  "id": 80,
+  "user": {
+    "login": "octocat",
+    "id": 1,
+    "avatar_url": "https://github.com/images/error/octocat_happy.gif",
+    "gravatar_id": "",
+    "url": "https://api.github.com/users/octocat",
+    "html_url": "https://github.com/octocat",
+    "followers_url": "https://api.github.com/users/octocat/followers",
+    "following_url": "https://api.github.com/users/octocat/following{/other_user}",
+    "gists_url": "https://api.github.com/users/octocat/gists{/gist_id}",
+    "starred_url": "https://api.github.com/users/octocat/starred{/owner}{/repo}",
+    "subscriptions_url": "https://api.github.com/users/octocat/subscriptions",
+    "organizations_url": "https://api.github.com/users/octocat/orgs",
+    "repos_url": "https://api.github.com/users/octocat/repos",
+    "events_url": "https://api.github.com/users/octocat/events{/privacy}",
+    "received_events_url": "https://api.github.com/users/octocat/received_events",
+    "type": "User",
+    "site_admin": false
+  },
+  "body": "Here is the body for the review.",
+  "commit_id": "ecdd80bb5713475d7ba9641ffaa4d7d2c19d3f3091",
+  "state": "PENDING",
+  "html_url": "https://github.com/octocat/Hello-World/pull/1347#pullrequestreview-80",
+  "pull_request_url": "https://api.github.com/repos/octocat/Hello-World/pulls/1347",
```

Comment:
Altered the example to use `1347` instead of `12` for convenience to allow use of `url_for` without a replace. Can switch this around instead to ensure it matches the documentation exactly at https://developer.github.com/v3/pulls/reviews/#submit-a-pull-request-review

Reply:
That's fine. I've done the same thing in the pats as well. :+1: